openspec/changes/add-eve-dynamic-workflow-orchestration/design.md

```
@@ -0,0 +1,100 @@
+# Design: Eve Dynamic Workflow Orchestration
+
+## Context
+
+#434 depends on the governance and budget controls that decide whether work may occur, the runtime that owns
```

markdownlint-cli2 (0.23.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)